api/urls.py

```diff
@@ -40,6 +40,7 @@
     url(r'rss/?', views.CreativeWorksRSS(), name='rss'),
     url(r'atom/?', views.CreativeWorksAtom(), name='atom'),
     url(r'userinfo/?', views.ShareUserView.as_view(), name='userinfo'),
+    url(r'v1data/?', views.V1DataView.as_view(), name='v1data'),
```

Comment:
Do you mean change the name so it's the same as the old one? Or would mapping the new to the old involve something more?